apps/web/src/pages/ApplicantDashboardPage/components/CareerDevelopmentTaskCard.tsx

```diff
@@ -96,13 +96,6 @@ const CareerDevelopmentTaskCard = ({
         </>
       ),
     },
-    {
-      key: "career-planning-key",
-      type: "link",
-      href: paths.employeeProfile(),
-      color: "secondary",
-      children: <>{intl.formatMessage(commonMessages.careerPlanning)}</>,
-    },
```

Comment:
Per @JoshBeveridge 
A second link to career planning section underneath functional communities is no longer desired